From 11305c414269379a3ba60cba081f9cf5e569285c Mon Sep 17 00:00:00 2001
From: Wolfgang Grandegger <wg@aries-embedded.de>
Date: Wed, 2 Jan 2019 11:09:05 +0100
Subject: [PATCH 11/11] mcvevk: fix device tree to work with DM serial

Device trees need to have the serial console device available
before relocation and require a stdout-path in chosen at least
for SPL to have a console. This has been done for the other
socfpga boards by commit 79a436d56848c4213c1452ca60211568b0c8b31a.

Signed-of-by: Wolfgang Grandegger <wg@aries-embedded.de>
---
 arch/arm/dts/socfpga_cyclone5_mcvevk.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/dts/socfpga_cyclone5_mcvevk.dts b/arch/arm/dts/socfpga_cyclone5_mcvevk.dts
index 65bec15..9d9bbcf 100644
--- a/arch/arm/dts/socfpga_cyclone5_mcvevk.dts
+++ b/arch/arm/dts/socfpga_cyclone5_mcvevk.dts
@@ -11,6 +11,7 @@
 
 	chosen {
 		bootargs = "console=ttyS0,115200";
+		stdout-path = "serial0:115200n8";
 	};
 
 	aliases {
@@ -56,3 +57,7 @@
 	disable-over-current;
 	status = "okay";
 };
+
+&uart0 {
+        u-boot,dm-pre-reloc;
+};
-- 
1.9.1

